sktime_dl/deeplearning/encoder/_classifier.py

```diff
@@ -24,46 +25,41 @@ class EncoderClassifier(BaseDeepClassifier, EncoderNetwork):
        pages={120},
        year={2018}
     }
+
+    :param nb_epochs: int, the number of epochs to train the model
+    :param batch_size: int, specifying the length of the 1D convolution
+     window
+    :param callbacks: list of tf.keras.callbacks.Callback objects
+    :param random_seed: int, seed to any needed random actions
+    :param verbose: boolean, whether to output extra information
+    :param model_name: string, the name of this model for printing and
+     file writing purposes
+    :param model_save_directory: string, if not None; location to save
+     the trained keras model in hdf5 format
     """
 
     def __init__(
-        self,
-        nb_epochs=100,
-        batch_size=12,
-        callbacks=None,
-        random_seed=0,
-        verbose=False,
-        model_name="encoder",
-        model_save_directory=None,
+            self,
+            nb_epochs=100,
+            batch_size=12,
+            callbacks=None,
+            random_seed=0,
+            verbose=False,
+            model_name="encoder",
+            model_save_directory=None,
     ):
         super().__init__(
```

Comment:
As per comment above, 'cnn/_classifier.py', does not use `super` but this file, encoder/_classifier.py, does use `super`. Which is preferred?